### PR TITLE
[vsts release definition] Fixing helptext

### PR DESCRIPTION
**Pull request by ds-ms**

### CODE REVIEW

**Comment by geverghe** [comment]
The example for "starting with" search has been removed. However, the instruction still says "Limit results to definitions with this name **_or starting with this name._**" The highlighted section has to be removed if not supported.

**Comment by ds-ms** [comment]
Starting with this name actually is apt.
```
PS C:\Users\desattir> vsts release definition list --name N -o table
  ID  Name                         CreatedBy         Created On
----  ---------------------------  ----------------  --------------------------------
   7  aks - Service account tests  Deepak Sattiraju  2018-11-06T06:30:44.773000+00:00
   2  New release pipeline         Deepak Sattiraju  2018-08-28T10:40:51.810000+00:00
   3  New release pipeline (1)     Deepak Sattiraju  2018-10-09T08:24:02.200000+00:00
   4  New release pipeline (2)     Deepak Sattiraju  2018-10-12T08:38:44.370000+00:00
   5  New release pipeline (3)     Deepak Sattiraju  2018-10-22T04:31:57.250000+00:00
   6  New release pipeline (4)     Deepak Sattiraju  2018-11-01T08:41:50.893000+00:00
   8  New release pipeline (5)     Deepak Sattiraju  2018-11-15T06:08:04.463000+00:00
```

**Comment by ds-ms** [comment]
@geverghe This is the function being used: https://github.com/Microsoft/azure-devops-python-api/blob/dev/vsts/vsts/release/v4_0/release_client.py#L330 

It's just a different convention, `FabCi` is equivalent to `*FabCi*` for this API call.

File: src/common_modules/vsts-cli-release-common/vsts/cli/release/common/release_definition.py [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
 def release_definition_list(name=None, top=None, team_instance=None, project=None, artifact_source_id=None, detect=None):
     """List release definitions.
-    :param name: Limit results to definitions with this name or starting with this name. Examples: "FabCI" or "Fab*"
+    :param name: Limit results to definitions with this name or starting with this name. Examples: "FabCI"
```
Comment (atbagga) [inline src/common_modules/vsts-cli-release-common/vsts/cli/release/common/release_definition.py]:
```suggestion
    :param name: Limit results to definitions with this name or starting with this name. Example: "FabCI"
```

**Pull request merged**